workflow/scripts/de_analysis.py

```
@@ -36,29 +38,33 @@
 # next we filter out counts, with counts lower than 10
-genes_to_keep = counts_df.columns[counts_df.sum(axis=0) >= snakemake.config["mincount"]]
+genes_to_keep = counts_df.columns[
+    counts_df.sum(axis=0) >= snakemake.config["deseq2"]["mincount"]
+]
 counts_df = counts_df[genes_to_keep]
 
 dds = DeseqDataSet(
     counts=counts_df,
     metadata=metadata,
-    design_factors=snakemake.config["design_factors"],
+    design_factors=snakemake.config["deseq2"]["design_factors"],
```

Ruff

Undefined name `snakemake`

(F821)